app/scripts/views/sign_up.js

```diff
@@ -225,6 +227,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
       return this._isMailcheckEnabledValue;
     },
+    _wasMailcheckUseful: function () {
+      var email = this.$el.find('.email');
```

Comment:
I'm accessing `.data` below of the email input. 
If I use `getElementValue` then it won't be useful and I have to query for the element twice